docs/iviewer.rst

```diff
@@ -69,6 +71,12 @@ default viewer for OMERO.web.
 
    .. image:: images/Iviewer2.png
 
+#. Still staying in the "Settings" tab in the right-hand side of OMERO.iviewer, click on the checkbox next to the "Histogram" word |image0| near the top.
+
+#. A histogram of the first channel which is on will appear. Note that when you move the rendering settings slider beneath the histogram, the histogram reverst to showing the channel you just manipulated.
```

Comment:
reverst -> reverts

Reply:
@will-moore Fixed in https://github.com/ome/omero-guide-iviewer/pull/2/commits/c3a35413b99e68c242e47e58a64965841046891a